test/index.spec.ts

```diff
@@ -78,16 +78,11 @@ describe("addForwarder", () => {
       handler: "hello.handler",
     });
     addForwarder(stack, [nodeLambda, pythonLambda], "forwarder-arn");
-    const nodeLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(
-      nodeLambda.functionArn,
-      "forwarder-arn",
-    );
-    const pythonLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(
-      pythonLambda.functionArn,
-      "forwarder-arn",
-    );
-    expect(nodeLambda.logGroup.node.tryFindChild(nodeLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
-    expect(pythonLambda.logGroup.node.tryFindChild(pythonLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
+
+    const subscriptionFilters = findDatadogSubscriptionFilters(stack);
+    expect(nodeLambda.logGroup.node.tryFindChild(subscriptionFilters[0].id)).not.toBeUndefined();
+    expect(pythonLambda.logGroup.node.tryFindChild(subscriptionFilters[1].id)).not.toBeUndefined();
+    expect(subscriptionFilters[0].destinationArn).toEqual(subscriptionFilters[1].destinationArn);
   });
 
   it("Throws an error when a customer redundantly calls the addLambdaFunctions function on the same lambda function(s) and forwarder", () => {
```

Comment:
disregard this [comment](https://github.com/DataDog/datadog-cdk-constructs/pull/30/files#r616051414) for this test